go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/chainguard-dev/terraform-infra-common
 
 go 1.21.6
 
+replace cloud.google.com/go/bigquery => github.com/imjasonh/google-cloud-go/bigquery v0.0.0-20240311234559-1dc0a176b346
```

Comment:
is the plan to upstream your changes or to depend on your fork moving forward
please add a comment